app/javascript/listings/__tests__/ListingFiltersCategories.test.jsx

```diff
@@ -19,83 +20,65 @@ describe('<ListingFiltersCategories />', () => {
-      expect(listingsLink.attr('className')).toBe('');
-      expect(listingsLink.text()).toBe('all');
+      const allLink = getByText(/all/i);
+      expect(allLink.getAttribute('href')).toBe('/listings');
+      expect(allLink.textContent).toBe('all');
     });
 
-    it('When there\'s no category, the className of the listings link should be "selected"', () => {
+    it('should be "selected" when there is no category selected', () => {
       const propsWithoutCategory = { ...getProps(), category: '' };
-      const contextWithoutCategory = renderListingFilterCategories(
+      const { getByTestId } = renderListingFilterCategories(
         propsWithoutCategory,
       );
-
-      const listingsLink = contextWithoutCategory.find('#listings-link');
-      expect(listingsLink.attr('className')).toBe('selected');
+      getByTestId('selected');
     });
 
-    it('Should render a link and a message relative to new listing', () => {
-      const newListingLink = context.find('#listings-new-link');
-
-      expect(newListingLink.attr('href')).toBe('/listings/new');
-      expect(newListingLink.attr('className')).toBe('listing-create-link');
-      expect(newListingLink.text()).toBe('Create a Listing');
+    it('should render a create listing link', () => {
+      const { getByText } = renderListingFilterCategories();
+      const createListing = getByText(/create a listing/i);
+      expect(createListing.getAttribute('href')).toContain('/listings/new');
     });
 
-    it('Should render a link and a message relative to dashboard', () => {
-      const dashboardLink = context.find('#listings-dashboard-link');
-
-      expect(dashboardLink.attr('href')).toBe('/listings/dashboard');
-      expect(dashboardLink.attr('className')).toBe('listing-create-link');
-      expect(dashboardLink.text()).toBe('Manage Listings');
+    it('should render a manage listings link', () => {
+      const { getByText } = renderListingFilterCategories();
+      const manageListing = getByText(/manage listings/i);
+      expect(manageListing.getAttribute('href')).toContain('/listings/dashboard');
     });
   });
 
-  describe('Should render categories links', () => {
-    const context = renderListingFilterCategories();
-    it('Should render the categories name and their respective links', () => {
+  describe('should render all the categories links', () => {
+    it('should render the categories name and their respective links', () => {
+      const { getByText } = renderListingFilterCategories();
       categories.forEach((category) => {
-        const categoryLink = context.find(`#category-link-${category.slug}`);
-        expect(categoryLink.attr('href')).toBe(`/listings/${category.slug}`);
-        expect(categoryLink.text()).toBe(category.name);
+        const categoryLink = getByText(`${category.name}`);
+        expect(categoryLink.getAttribute('href')).toBe(`/listings/${category.slug}`);
+        expect(categoryLink.textContent).toBe(category.name);
```

Comment:
Just to be consistent with the other tests you have and I have, use `toEqual` instead of `toBe`, even if it is a primitive type (string).

Reply:
✅